webv2/updateTableSchema/reviewrenameColumn.go

```diff
@@ -0,0 +1,216 @@
+package updateTableSchema
+
+import (
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/cloudspannerecosystem/harbourbridge/spanner/ddl"
+)
+
+// reviewRenameColumn review  rename Columnname in schmema.
+func reviewRenameColumn(newName, table, colName string, Conv *internal.Conv, interleaveTableSchema []InterleaveTableSchema) []InterleaveTableSchema {
+
+	sp := Conv.SpSchema[table]
+
+	columnId := sp.ColDefs[colName].Id
+
+	// update foreignKey relationship Table column names
```

Comment:
The comments are not clear. Please rephrase. Same for others

Reply:
Updated